openslides/utils/cache.py

```diff
@@ -296,7 +297,10 @@ def get_redis_connection() -> Any:
 
 if use_redis_cache():
     websocket_user_cache = RedisWebsocketUserCache()  # type: BaseWebsocketUserCache
-    restricted_data_cache = RestrictedDataCache()  # type: Union[RestrictedDataCache, DummyRestrictedDataCache]
+    if settings.get('DISABLE_USE_CACHE', False):
```

Comment:
Typo `USER`

Reply:
Is it also possible to to it as shown [here](https://docs.djangoproject.com/en/1.11/topics/settings/#using-settings-in-python-code)?

Reply:
Done.